workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -175,19 +189,17 @@ def _get_metric(self, metric_type: str, variable: str, run: str) -> pd.Series:
         return _metrics[_metrics.run == run]
 
 
-def load_metrics(fs, bucket) -> pd.DataFrame:
+def load_metrics(rundirs) -> pd.DataFrame:
```

Comment:
I agree rundirs is a bad name, but I'll punt on this for now, since this naming is somewhat ubiquitous, and it will take some time to track down all the instances of `rundirs` and fix the tests accordingly. I think this PR does take a step in the right direction.

Reply:
Okay sounds good.